src/components/todo/TodoInput.js

```diff
@@ -1,11 +1,38 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
 import styled from 'styled-components';
 
+const InputWrapper = styled.div`
```

Comment:
```suggestion
const InputWrapper = styled.form`
```
위의 변경점을 반영했을 때의 장점과 단점에 대해서 생각해보세요~ 🤔